airbyte-workers/src/main/java/io/airbyte/workers/temporal/SyncWorkflow.java

```diff
@@ -72,20 +74,28 @@ StandardSyncOutput run(JobRunConfig jobRunConfig,
         .setRetryOptions(TemporalUtils.NO_RETRY)
         .build();
 
-    private final SyncActivity activity = Workflow.newActivityStub(SyncActivity.class, options);
+    private final ReplicateActivity replicateActivity = Workflow.newActivityStub(ReplicateActivity.class, options);
```

Comment:
Should be:
- ReplicateActivity & NormalizeActivity
- ReplicationActivity & NormalizationActivity